### DISCUSSION TITLE
More Features

### DISCUSSION THREAD

**Discussion by rnwonder** [rnwonder/solid-date-picker#4] [Ideas]
If you have any feature you would like let us know

**Comment by tsdevau** [comment]
Add a prop for weekStartDay.  Can use the Date day object index range.  For example:

```tsx
<DatePicker 
  value={value}
  locale="en-AU"
  weekStartDay={1}
/>
```

Should then display the calendar grid as:

| Mon | Tue | Wed | Thu | Fri | Sat | Sun |
|----:|----:|----:|----:|----:|----:|----:|
|     |  1  |  2  |  3  |  4  |  5  |  6  |
|  7  |  8  |  9  | 10  | 11  | 12  | etc |

**Comment by rnwonder** [comment]
Oh cool

**Comment by rnwonder** [comment]
@ Implemented. Please Upgrade to the latest version. You can also check it out on the [Demo](https://stackblitz.com/edit/solidjs-templates-dof6jl?file=package.json,src%2FApp.tsx). Under Customize Dates no 4 and 5

**Reply by tsdevau** [reply]
Awesome, works great for me!

**Reply by rnwonder** [reply]
> Awesome, works great for me!

You might want to upgrade again added class props few minutes ago for easier customization. You can check example 4 on the demo
